source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -0,0 +1,306 @@
+         :language: csharp
+
+   .. tab:: Synchronous
+      :tabid: change-stream-sync
+
+      .. literalinclude:: /includes/code-examples/change-streams/change-streams.cs
+         :start-after: start-change-stream-pipeline
+         :end-before: end-change-stream-pipeline
+         :language: csharp
+
+To learn more about modifying your change stream output, see the
+:manual:`Modify Change Stream Output
+</changeStreams/#modify-change-stream-output>` section in the {+mdb-server+}
+manual.
+
+Modify ``Watch()`` Behavior
+---------------------------
+
+The ``Watch()`` and ``WatchAsync()`` methods accept optional parameters, which represent
+options you can use to configure the operation. If you don't specify any
+options, the driver does not customize the operation.
+
+The following table describes the options you can set to customize the behavior
+of ``Watch()`` and ``WatchAsync()``:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``FullDocument``
+     - | Specifies whether to show the full document after the change, rather
+         than showing only the changes made to the document. To learn more about
+         this option, see :ref:`csharp-change-stream-pre-post-image`.
+
+   * - ``FullDocumentBeforeChange``
+     - | Specifies whether to show the full document as it was before the change, rather
+         than showing only the changes made to the document. To learn more about
+         this option, see :ref:`csharp-change-stream-pre-post-image`.
+    
+   * - ``ResumeAfter``
+     - | Directs ``Watch()`` or ``WatchAsync()`` to resume returning changes after the
+         operation specified in the resume token.
+       | Each change stream event document includes a resume token as the ``_id``
+         field. Pass the entire ``_id`` field of the change event document that
+         represents the operation you want to resume after.
+       | ``ResumeAfter`` is mutually exclusive with ``StartAfter`` and ``StartAtOperationTime``.
+
+   * - ``StartAfter``
+     - | Directs ``Watch()`` or ``WatchAsync()`` to start a new change stream after the
+         operation specified in the resume token. Allows notifications to
+         resume after an invalidate event.
+       | Each change stream event document includes a resume token as the ``_id``
+         field. Pass the entire ``_id`` field of the change event document that
+         represents the operation you want to resume after.
+       | ``StartAfter`` is mutually exclusive with ``ResumeAfter`` and ``StartAtOperationTime``.
+       
+   * - ``StartAtOperationTime``
+     - | Directs ``Watch()`` or ``WatchAsync()`` to return only events that occur after the
+         specified timestamp.
+       | ``StartAtOperationTime`` is mutually exclusive with ``ResumeAfter`` and ``StartAfter``.
+
+   * - ``MaxAwaitTime``
+     - | The maximum amount of time, in milliseconds, the server waits for new
```

Comment:
S: since the first few descriptions start with a verb, it would be good to keep that consistent throughout the descriptions

```suggestion
     - | Specifies the maximum amount of time, in milliseconds, the server waits for new
```

Reply:
Applies to all